tests/pfcwd/test_pfc_config.py

```diff
@@ -268,3 +329,27 @@ def test_high_restore_time_cfg(self, duthosts, rand_one_dut_hostname):
         """
         duthost = duthosts[rand_one_dut_hostname]
         self.execute_test(duthost, "pfc_wd_high_restore_time", None, [CONFIG_TEST_EXPECT_INVALID_RESTORE_TIME_RE], True)
+
+@pytest.mark.usefixtures('mg_cfg_setup')
+class TestDefaultPfcConfig(object):
+    def test_default_cfg_after_load_mg(self, duthosts, rand_one_dut_hostname):
+        """
+        Tests for checking if pfcwd gets started after load_minigraph
+
+        Args:
+            duthost(AnsibleHost): instance
+
+        Returns:
+            None
+        """
+        duthost = duthosts[rand_one_dut_hostname]
+
+        duthost.shell('config load_minigraph -y &>/dev/null', executable="/bin/bash")
+        # sleep 20 seconds to make sure configuration is loaded
+        time.sleep(20)
```

Comment:
load minigraph using this func "config_reload(duthost, config_source='minigraph')"

Reply:
Done.